itl/command-plugins-manubulon.conf

```diff
@@ -183,6 +183,10 @@ object CheckCommand "snmp-interface" {
 		"-e" = {
 			set_if = "$snmp_interface_errors$"
 		}
+		"-i" = {
+			set_if = "$snmp_interface_inverse$"
+		}
+		"-r" = {
```

Comment:
Seems like you added the switch (-r) a second time in the file. Before someone can merge this you should fix this